inkstitch.py

```diff
@@ -31,55 +105,96 @@
 from lib.i18n import _
 from lib.utils import restore_stderr, save_stderr
 
-# ignore warnings in releases
-if getattr(sys, 'frozen', None):
+# file DEBUG exists next to inkstitch.py - enabling debug mode depends on value of debug_type in DEBUG file
+if debug_type != 'none':
+    debug_file = ini.get("DEBUG","debug_file", fallback="debug.log")
```

Comment:
Let's make an option for `debug.svg` too for consistency.  `debug.svg` is really useful for debugging graph-related things in auto-fill and the like.

Reply:
I've added to DEBUG.ini
```
;;; debug file for graph related things, default: debug.svg
; debug_graph_file = debug.svg
```